### PR TITLE
nft stuff

### PR DESCRIPTION
**Pull request by zqiang-bolt**

### CODE REVIEW

**Review by DerrickGHW — APPROVED** [review]

File: contracts/MyNFT.sol [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+
+contract MyNFT is ERC721URIStorage, Ownable {
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
+
+    constructor() ERC721("MyNFT", "NFT") {}
```
Comment (sepandar-sepehr) [inline contracts/MyNFT.sol:14]: Do you want to rename this?

File: scripts/deploy.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+async function main() {
+    const MyNFT = await ethers.getContractFactory("MyNFT")
```
Comment (sepandar-sepehr) [inline scripts/deploy.js:2]: Same name change here.

File: contracts/MyNFT.sol (hunk 1 above)
Comment (sepandar-sepehr) [reply contracts/MyNFT.sol:14]: From the blog: 
> In our ERC-721 constructor, you’ll notice we pass 2 strings, “MyNFT” and “NFT.” The first variable is the smart contract’s name, and the second is its symbol. You can name each of these variables whatever you wish!

File: scripts/mint-nft.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+async function sendToken() {
+    const nonce = await web3.eth.getTransactionCount(PUBLIC_KEY, "latest") //get latest nonce
+
+    //the transaction
+    const tx = {
+        from: PUBLIC_KEY,
+        to: contractAddress,
+        nonce: nonce,
+        gas: 500000,
+        data: nftContract.methods.safeTransferFrom(PUBLIC_KEY, "0x7aaa2785baaf248b91fbc64b70280bcf3ddd4e09", "1").encodeABI(),
```
Comment (sepandar-sepehr) [inline scripts/mint-nft.js]: `0x7aaa2785baaf248b91fbc64b70280bcf3ddd4e09` is wallet ID? Want to make it a const?

**Pull request merged**